quick/commands/streams_app.py

```diff
@@ -58,6 +61,12 @@ def add_args(self, parser: ArgumentParser, required: ArgumentGroup, optional: Ar
             help="Docker image tag",
             required=True,
         )
+        optional.add_argument(
+            "--image-pull-secret",
+            metavar="IMAGE_PULL_SECRET",
+            type=str,
+            help="A secret in a string format for pulling an image from a private registry",

```

Comment:
The term "in a string" format is still too abstract. Maybe change this to something like this:
```
The name of the image pull secret
```